internal/sandbox/sandbox.go

```diff
@@ -428,6 +438,14 @@ func (s *podmanSandbox) Run(args ...string) (*RunResult, error) {
 	}
 	errWriter := io.MultiWriter(errWriters...)
 
+	log.Debug("upload certs to container")
+	uploadCmd := s.UploadFileToContainer("/proxy/certs/ca.crt", "/usr/local/share/ca-certificates/ca.crt") //upload certs to container
```

Comment:
Rather than hardcoding the input cert location, I suggest creating an `Option` for passing "/proxy/certs/ca.cert" in to the sandbox.

Reply:
Please take a look at the new `Copy` option that has been added.